test/Makefile

```diff
@@ -11,3 +13,23 @@ else
 	$(QUIET)./ginkgo.sh
 endif
 
+.PHONY: kind-init
+kind-init:
+	@echo "Install kind cluster..."
+	@kube_proxy_mode=$(KUBE_PROXY_MODE) ip_family=$(IP_FAMILY) singe_node=$(SINGE_NODE) kind_image_tag=$(KIND_IMAGE_TAG) disable_default_cni=$(DISABLE_DEFAULT_CNI) p2ctl -t ../tools/yamls/kind-config.tmpl > ../tools/yamls/kind-config.yaml
+	@cat ../tools/yamls/kind-config.yaml
+	kind create cluster --config ../tools/yamls/kind-config.yaml --name $(CLUSTER_NAME)
+	kubectl apply -f ../tools/yamls/cni-install.yaml
+ifneq ($(INSTALL_MACVLAN),)
+	@echo "Install macvlan + whereabouts"
+	@bash ../tools/scripts/install-macvlan.sh
+endif
+	kubectl get nodes
```

Comment:
kubectl get nodes &>/dev/null || echo "failed to setup  cluster"

Reply:
Done